<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appPreferencesDialog</class>
 <widget class="QDialog" name="appPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Умолчания</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkCreateNomenclatureDuringImport">
     <property name="text">
      <string>Создавать номенклатуру при импорте</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLogErrors">
     <property name="text">
      <string>Формировать файл ошибок загрузки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLogSuccess">
     <property name="text">
      <string>Формировать файл успешной загрузки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbService">
     <property name="title">
      <string>Сервис</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkAutoImport">
        <property name="text">
         <string>Автозапуск импорта</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblScanDir">
          <property name="text">
           <string>Директория обмена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtScanDir"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnSelectScanDir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblScanPeriod">
          <property name="text">
           <string>Интервал проверки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="edtScanPeriod">
          <property name="suffix">
           <string>сек</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>60000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lblOrganisation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;ЛПУ</string>
       </property>
       <property name="buddy">
        <cstring>cmbOrganisation</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CPolyclinicComboBox" name="cmbOrganisation"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectOrganisation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPolyclinicComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Orgs/OrgComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>appPreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
